frappe/modules/utils.py

```diff
@@ -158,6 +158,10 @@ def _insert(data):
 			if doc_type == doctype or not os.path.exists(os.path.join(folder, scrub(doc_type) + ".json")):
 				sync_single_doctype(doc_type)
 
+	if not frappe.db.exists("DocType", doctype):
+		print(_("DocType {0} does not exist").format(doctype))
```

Comment:
Not so easy to guess from where this message is coming from. 


Message should indicate that we are skipping syncing of some fixture file?